tensorflow_addons/activations/snake.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+import tensorflow as tf
+
+from tensorflow_addons.utils import types
+
+
+@tf.keras.utils.register_keras_serializable(package="Addons")
+def snake(inputs: types.TensorLike, freq: types.Number = 1) -> tf.Tensor:
+    """Snake activation to learn periodic functions.
+
+    https://arxiv.org/abs/2006.08195
+    """
```

Comment:
could we add the arguments doc here?

Reply:
I was trying, but the original academic paper is quite obscure about explanations. So it could has end-up with words like 'frequency is a frequency' and other obvious things such as 'input is an input'. If it is OK anyway, no problems, I'll add.

Reply:
Got it. Maybe we can have something like

```
      inputs: Input tensor.
      frequency: A scalar, frequency of the periodic part.
```